pkgs/applications/misc/calibre/default.nix

```diff
@@ -42,9 +43,20 @@ mkDerivation rec {
     # the unrar patch is not from debian
   ] ++ lib.optional (!unrarSupport) ./dont_build_unrar_plugin.patch;
 
+  escaped_pyqt5_dir = builtins.replaceStrings ["/"] ["\\/"] (toString python3Packages.pyqt5);
+  platform_tag =
+    if stdenv.hostPlatform.isDarwin then
+      ''\"WS_MACX\"''
```

Comment:
Did you think about leaving it here unescaped and putting the escapes into the sed? Might be a little bit easier to read.

Reply:
Changed, indeed looks better.